### ISSUE TITLE
App access info

### ISSUE THREAD

**Issue by pmsousa** [LanceMcCarthy/MvpApi#44]
Hi,

Every time I open the App it asks for permissions to access my info. This should be done only once on first use and again if anything changed on permission needed by the app. :)

PS: @LanceMcCarthy - great job with the App. Kudos 🥇

**Comment by LanceMcCarthy** [comment]
The max session time is 60 minutes, the API will enforce re-authentication. There is a way to avoid that screen everytime by using the OAuth refresh token. However, I'm horrible at OAuth and don't know how to use it :) 

If you do, here's my open Issue for that task  https://github.com/LanceMcCarthy/MvpApi/issues/16

**Comment by LanceMcCarthy** [comment]
@pmsousa This has been addressed in the latest update (1.7). Get the update using the link (I'm only rolling out 20% at a time and the link will force the update) https://www.microsoft.com/store/apps/9NRXNX3WLH77

**Comment by pmsousa** [comment]
> @pmsousa This has been addressed in the latest update (1.7). Get the update using the link (I'm only rolling out 20% at a time and the link will force the update) https://www.microsoft.com/store/apps/9NRXNX3WLH77

Hi @LanceMcCarthy,

My app is "stuck" at version 1.6.0 and there's no updates on the Store... Even from the link, it sees I have it installed and changes the option from "Get" to "Launch".

**Comment by LanceMcCarthy** [comment]
Hmm, okay that seems like a bug in DevCenter. Let me increase the rollout %.

**Comment by LanceMcCarthy** [comment]
@pmsousa Okay, I finalized the roll out. IT might take a few minutes, but try the link again.

Note that the very first time you launch the app, you might get an error message. If this happens just close the app and then reopen it. It should automatically log you in.

**Comment by pmsousa** [comment]
@LanceMcCarthy, I'll grab a cup of coffee and try again...

PS: I just convinced one of my colleagues to try your app also... ;)

**Comment by LanceMcCarthy** [comment]
Awesome let me know how it goes. 

Note: I have big plans for v 2.0, it'll be a Fluent redesign as well as new functional features, like in DataGrid editing.

**Comment by pmsousa** [comment]
And it works!!! 🥇 

PS: Have some ideas for your next version...

**Comment by LanceMcCarthy** [comment]
YES! Thank you for the feedback.

For the ideas, don't hesitate to add a new Issue and mark it with Feature Request, That way I can work on them individually and you can track their progress..
